frontend/src/components/RequestDialog.vue

```diff
@@ -162,7 +162,7 @@ const submitRequestTransaction = async (formResult: {
 };
 
 watch(ethereumProvider.value.chainId, async () => {
-  await executeGetFee();
```

Comment:
Why got this removed? 🤔

Reply:
It's called anyway after reloading the screen `line 167`